### ISSUE TITLE
Validate Method not catching the error when no api key is passed

### ISSUE THREAD

**Issue by ros3bud** [activeprospect/leadconduit-integration-national-renewable-energy-lab#5]
It is just checking the whether the credential has any value - so it returns true because the masked and valid properties are returned. However, it needs to more specifically look for a credential value and stop the lead from being sent to NREL when it is not present.

**Comment by alexkwolfe** [comment]
```coffeescript
return 'NREL API key must not be blank' unless vars.nrel?.api_key?.valueOf()
```

**Comment by ros3bud** [comment]
Yes - I have verified that this will work. :)

**Comment by ros3bud** [comment]
Resolved in 0.0.4
